packages/core/src/modules/proofs/ProofState.ts

```diff
@@ -12,4 +12,5 @@ export enum ProofState {
   PresentationReceived = 'presentation-received',
   Declined = 'declined',
   Done = 'done',
+  None = 'none',
```

Comment:
I'm not sure if state `none` is a good representation of the state. Shouldn't this be more something like `error`?

@jakubkoci @JamesKEbert any insights?